cds_ils/migrator/document_requests/api.py

```diff
@@ -32,8 +32,8 @@ def migrate_document_request(record):
 
     if record["status"] == "proposal-put aside":
         state = "REJECTED"
-        reject_reason = record["library_notes"]
-        new_docreq.update(state=state, reject_reason=reject_reason)
+        new_docreq.update(state=state,
+                          reject_reason="MIGRATED_UNKNOWN")
```

Comment:
This is was called `MIGRATED` only if not mistaken in the other PR, but I suggested to use something else, e.g. `OTHER`.